src/components/MaterialUI/DataDisplay/TableCell.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef } from "react";
+import TableCellMui from "@material-ui/core/TableCell";
+import { makeStyles } from "@material-ui/core/styles";
+import classNames from "classnames";
+import Tooltip from "./Tooltip";
+
+const useStyles = makeStyles(theme => ({
+  cellRoot: {
+    maxWidth: theme.spacing(15),
+    padding: props => theme.spacing(props.cellPaddingInPixels / 10)
+  }
+}));
+
+const TableCell = ({ cell }) => {
+  const cellPaddingInPixels = 16;
+
+  const classes = useStyles({ cellPaddingInPixels });
+
+  const ref = useRef(null);
+
+  const [clientWidth, setWidth] = React.useState(0);
+
+  useEffect(() => {
+    setWidth(ref.current.clientWidth - 2 * cellPaddingInPixels); // left and right padding
+  }, []);
```

Comment:
You need to put some stuff between [], ie, "ref" and "cellPaddingInPixels".

Also, you would need to put a comment about eslint to get rid of the annoying warning about "setWidth".

Reply:
Well, I don't see esling warning but usually, it compains about it, strange. Run "npm run lint" locally to see if you have any. At the same time, could you remove those from MultipleLineText component please ?